get-started/part2.md

```diff
@@ -141,7 +141,7 @@ Now that you have some source code and a Dockerfile, it's time to build your fir
 Make sure you're in the directory `node-bulletin-board/bulletin-board-app` in a terminal or PowerShell using the `cd` command. Let's build your bulletin board image:
 
 ```script
-docker image build -t bulletinboard:1.0 .
+docker build -t bulletinboard:1.0 .
```

Comment:
Do we want to make this `--tag` instead of `-t`?

Reply:
yeah probably a good idea, we mostly use the long form flags as they are less confusing